polls/views.py

```diff
@@ -38,6 +38,17 @@ def get(self, request):
         return render(request, self.template_name, context)
 
 
+class AdminDashboardView(LoginRequiredMixin, CheckUserLockMixin, View):
+    template_name = "polls/admin_dashboard.html"
```

Comment:
CheckUserLlockMixin should have nothing to do here, the page should be assesible by admin users only, so I think a UserPassesTestMixin with test func returning user.is_admin would work the best for permission case and blocking non Admin users.

Reply:
Sure will take care of this

Reply:
Kindly resolve this once it's fixed